### PR TITLE
Implementation of mc_dropput

### PR DESCRIPTION
**Pull request by javiercorrea**
- Implemented mc_dropout parameters for the Dropout layer
  allowing to Dropout layer to drop inputs at prediction
  time

### CODE REVIEW

**Comment by alexandari** [comment]
I don't think the specialized layer is necessary. A simpler way to get uncertainty estimates is to set learning_phase = 1 as outlined in the FAQ: https://keras.io/getting-started/faq/#how-can-i-obtain-the-output-of-an-intermediate-layer

**Comment by gabrieldemarmiesse** [comment]
Even easier, when calling the normal dropout, you can do this:

`x = Dropout(0.5)(x, training=True)` and tada! The dropout layer will always behave as if it was in training mode.
